airbyte-integrations/connectors/source-northpass-lms/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-northpass-lms"
+description = "Source implementation for northpass-lms."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/northpass-lms"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+
+[[tool.poetry.packages]]
+include = "source_northpass_lms"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
```

Comment:
```suggestion
airbyte-cdk = "^4.3.0"
```

Reply:
We've published _a few_ new versions of the CDK —let's upgrade to the latest one!